agent/bench-scripts/pbench-run-benchmark

```diff
@@ -60,34 +60,34 @@ sub find_common_parameters {
     my %common_parameters;
     my $counter = 1;
     for my $iteration_params (@iterations) {
-	$iteration_params =~ s/^\s+(.+)\s+$/$1/;
+        $iteration_params =~ s/^\s+(.+)\s+$/$1/;
```

Comment:
Ugh, I'll move the white-spaces changes to another commit.

Reply:
We are so close to committing this code that it no longer seems worth the effort to factor this white-space change out.